src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -147,30 +146,11 @@ public final void nativeFinishLayoutOverride(){
 
 	protected abstract void onLayoutCore(boolean changed, int left, int top, int right, int bottom);
 
-	private boolean _isInOnLayoutCore;
-
 	protected final void onLayout(boolean changed, int left, int top, int right, int bottom)
 	{
 		if(!_unoLayoutOverride)
 		{
-			try
-			{
-				_isInOnLayoutCore = true;
-
-				_childrenNeedingForceLayout.clear();
-				onLayoutCore(changed, left, top, right, bottom);
-
-				if (!_childrenNeedingForceLayout.isEmpty()) {
-					for (int i = 0; i < _childrenNeedingForceLayout.size(); i++) {
-						_childrenNeedingForceLayout.get(i).forceLayout();
```

Comment:
@YGuerin - do you recall why this code was added? Was it something about VisibleBoundsPadding not updating properly in some circumstance? 

@NicolasChampagne - can you explain why you're removing it?

Reply:
After testing on devices without this code VisibleBoundsPadding was being updated correctly.

It was also causing an infinite loop for some listviews, making apps lag